Specification.md

```diff
@@ -95,29 +91,49 @@ SEQ-NO-TYPE = 204
 
 Nodes can either be in _Steady State_, or in _Suppression State_
 
-- _Steady State_: The sync group is in sync. Incoming Sync Interests carry the latest known state.
-- _Suppression State_: Incoming Sync Interests indicate a state inconsistency. The node tries reconciling the inconsistency by emitting a up-to-date Sync Interest. Use a suppression timer before sending to prevent flooding.
+- _Steady State_: The Sync group is synchronized. Incoming Sync Interests carry the latest known state.
+- _Suppression State_: Incoming Sync Interests indicate a state inconsistency. The node tries to reconcile the inconsistency by emitting a up-to-date Sync Interest. Use a suppression timer before sending to prevent flooding.
+
+SVS utilizes a single Sync Interest timer. It can take on one of the two timeout values that are used by the protocol. The values required for the timer are as follows.
+
+- `PeriodicTimeout`: defaults to 30 seconds (±10% uniform)
+- `SuppressionPeriod`: defaults to 200ms
+- `SuppressionTimeout`: random value between 0 to `SuppressionPeriod` \
+  An exponential decay function SHOULD be used for the timeout value.
+  ```
+  c = SuppressionPeriod  // constant factor
+  v = random(0, c)       // uniform random value
+  f = 10.0               // decay factor
+  SuppressionTimeout = c * (1 - e^((v - c) / (c / f)))
```

Comment:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

Reply:
Not sure it works with the html stuff. Maybe next time